check_line_length.py

```diff
@@ -14,7 +14,7 @@ def check_url(line):
     """
     Check if there is a url present in the given line
     """
-    pattern = 'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\(\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
+    pattern = 'http[s]?://(?:[a-zA-Z]|[0-9]|[$-_@.&+]|[!*\\(\\), ]|(?:%[0-9a-fA-F][0-9a-fA-F]))+'
```

Comment:
Can you share why have you made this change? Was it causing a failure?

Reply:
Flake8 pointed this out. Moreover, there is no `\(` or `\)` character, so although python handles this, it's a good fix imho.